### PR TITLE
feat(tasks):  bash script type for script tasks

### PR DESCRIPTION
**Pull request by harmbrugge**
closes [#4293](https://github.com/molgenis/molgenis-emx2/issues/4293)

What are the main changes you did:
-  Added a bash script type for script task to execute bash script directly via scripts

how to test:
- Go to the preview server login as admin and go to Scripts
- create a script with type bash
- Command could be: echo "hello world!"
- execute it

todo:
- [ ] updated docs in case of new feature
- [x] added/updated tests
- [ ] added/updated testplan to include a test for this fix, including ref to bug using # notation

### CODE REVIEW

**Review by erikzwart — APPROVED** [review]
Functionally tested a simple bash script:

```bash
#!/bin/bash
string="Hello"
```
afbeelding

Which works as intended.

Also tested MOLGENIS_TOKEN:

```bash
#!/bin/bash
echo $MOLGENIS_TOKEN
```
afbeelding

Also works.

**Review by mswertz — CHANGES_REQUESTED** [review]
great feature

Indeed I would also like to see change to enum. 

Furthermore, we should update the UI so users can there also define it.
That might also influence the (not) changing the API signature comment.

(some more work but otherwise this feature is not accessible right?)

File: backend/molgenis-emx2-tasks/src/main/java/org/molgenis/emx2/tasks/ScriptTask.java [hunk 1]
```diff
@@ -190,6 +199,15 @@ public String getName() {
     return name;
   }
 
+  public ScriptTask type(String type) {
+    this.type = type;
+    return this;
```
[the remark this answers is not in the document]
Comment (mswertz) [reply backend/molgenis-emx2-tasks/src/main/java/org/molgenis/emx2/tasks/ScriptTask.java:204]: we follow the 'builder' pattern, see also for other setters.

File: backend/molgenis-emx2-tasks/src/main/java/org/molgenis/emx2/tasks/ScriptTask.java [hunk 2]
```diff
@@ -25,6 +25,7 @@
 public class ScriptTask extends Task {
   private static Logger logger = LoggerFactory.getLogger(ScriptTask.class);
   private String name;
+  private String type;
```
[the remark this answers is not in the document]
Comment (mswertz) [reply backend/molgenis-emx2-tasks/src/main/java/org/molgenis/emx2/tasks/ScriptTask.java:28]: agreed

**Review by mswertz — APPROVED** [review]

**Pull request merged**